Composer/packages/lib/ui-shared/src/components/ProvisionHandoff/styles.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { css } from '@emotion/core';
+import { FontWeights } from 'office-ui-fabric-react/lib/Styling';
+import { NeutralColors } from '@uifabric/fluent-theme';
+
+export const instructionStyles = css`
+  margin-bottom: 20px;
+`;
+export const copyPasteStyles = css`
+  background: ${NeutralColors.black};
+  color: ${NeutralColors.white};
+  padding: 15px;
+  max-height: 150px;
+  overflow: scroll;
+  white-space: pre;
+`;
+copyPasteStyles;
+
+export const dialog = {
```

Comment:
Consider naming this something more specific - dialog as an exported name could cause collisions.

Reply:
deprecated